SDK/AppCenterAuth/Microsoft.AppCenter.Auth.iOS/Auth.cs

```diff
@@ -10,8 +11,17 @@ public partial class Auth : AppCenterService
     {
         private static Task<SignInResult> PlatformSignInAsync()
         {
-            MSIdentity.SignIn(null);
-            return null;
+            var promise = new TaskCompletionSource<SignInResult>();
```

Comment:
Let's don't use "promise" in naming here

Reply:
changed to taskCompletionSource.